### PR TITLE
Fixed dimen.xml redundant values and removed redundant resource files

### LINKED ISSUE #453: Remove duplicate resources and dimens values
**Describe the bug**
Some resource files are redundant in the app codebase and not needed. 
Also, there are duplicate dimens values. Remove them and be sure to make changes wherever the duplicates were used throughout the codebase.

**Expected Behavior**
Remove duplicate resources and dimens values

**Steps to reproduce it**
Go through the resources and codebase of the app.

**Would you like to work on the issue?**
Issue for Codeheat.

### PR DESCRIPTION
**Pull request by CodeMonk263**
Fixes #453 

**Changes**: Removed redundant resource files and dimen.xml redundant values.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

File: app/src/main/res/raw/sample2.csv [hunk 1]
```diff
@@ -1,407 +0,0 @@
...
-2506,2019-08-16 09:36:04,4178.974359,22.67206349,22.67206349,,,,,,
-2507,2019-08-16 09:36:04,4166.666667,22.67206349,22.67206349,,,,,,
-2507,2019-08-16 09:36:04,4155.897436,22.67206349,22.67206349,,,,,,
-2508,2019-08-16 09:36:04,4162.564103,22.67206349,22.67206349,,,,,,
-2508,2019-08-16 09:36:04,4165.128205,22.67206349,22.67206349,,,,,,
-2509,2019-08-16 09:36:04,4167.692308,22.67206349,22.67206349,,,,,,
-2509,2019-08-16 09:36:04,4180,22.67206349,22.67206349,,,,,,
-2510,2019-08-16 09:36:04,4185.641026,22.67206349,22.67206349,,,,,,
-2510,2019-08-16 09:36:04,4181.538462,22.67206349,22.67206349,,,,,,
-2511,2019-08-16 09:36:04,4183,,,,,,,,
```
Comment (dipu989) [inline app/src/main/res/raw/sample2.csv:407]: These sample csv files are not redundant. These are used once app is installed and users get to work with these sample data. Don't delete them.

File: app/src/main/res/raw/sample3.csv [hunk 2]
```diff
@@ -1,450 +0,0 @@
...
-22.67208003,22.67208003,4157.948718,,,,,2409,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4157.948718,,,,,2410,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4148.205128,,,,,2410,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4152.307692,,,,,2411,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4163.076923,,,,,2411,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4167.179487,,,,,2412,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4165.641026,,,,,2412,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4159.487179,,,,,2413,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4157.948718,,,,,2413,2019-08-16 09:51:36,,
-22.67208003,22.67208003,4163.076923,,,,,2414,2019-08-16 09:51:36,,
```
Comment (dipu989) [inline app/src/main/res/raw/sample3.csv:450]: Same

File: app/src/main/res/raw/sample4.csv [hunk 3]
```diff
@@ -1,7169 +0,0 @@
...
-91932,2019-07-30 07:53:09,4174.871795,22.67210911,22.67210911,,,,,,
-91932,2019-07-30 07:53:09,4179.487179,22.67210911,22.67210911,,,,,,
-91933,2019-07-30 07:53:09,4167.179487,22.67210911,22.67210911,,,,,,
-91933,2019-07-30 07:53:09,4161.538462,22.67210911,22.67210911,,,,,,
-91933,2019-07-30 07:53:09,4173.333333,22.67210911,22.67210911,,,,,,
-91934,2019-07-30 07:53:09,4175.897436,22.67210911,22.67210911,,,,,,
-91934,2019-07-30 07:53:09,4165.128205,22.67210911,22.67210911,,,,,,
-91934,2019-07-30 07:53:09,4158.974359,22.67210911,22.67210911,,,,,,
-91935,2019-07-30 07:53:09,4162.564103,22.67210911,22.67210911,,,,,,
-91935,2019-07-30 07:
```
Comment (dipu989) [inline app/src/main/res/raw/sample4.csv:7169]: Same

File: app/src/main/res/raw/sample1.csv [hunk 4]
```diff
@@ -1,413 +0,0 @@
...
-2213,2019-08-14 16:37:13,4180,22.67214458,22.67214458,,,,,,
-2214,2019-08-14 16:37:13,4185.641026,22.67214458,22.67214458,,,,,,
-2214,2019-08-14 16:37:13,4181.538462,22.67214458,22.67214458,,,,,,
-2215,2019-08-14 16:37:13,4183.589744,22.67214458,22.67214458,,,,,,
-2215,2019-08-14 16:37:13,4187.692308,22.67214458,22.67214458,,,,,,
-2216,2019-08-14 16:37:13,4175.897436,22.67214458,22.67214458,,,,,,
-2216,2019-08-14 16:37:13,4166.153846,22.67214458,22.67214458,,,,,,
-2216,2019-08-14 16:37:13,4173.846154,22.67214458,22.67214458,,,,,,
-2217,2019-08-14 16:37:13,4175.897436,22.67214458,22.67214458,,,,,,
-2217,
```
Comment (dipu989) [inline app/src/main/res/raw/sample1.csv:413]: same

**Review by dipu989 — COMMENTED** [review]
Also squash all the commits into a single Commit.

**Comment by dipu989** [comment]
@CodeMonk263  You don't need to close this PR totally. If you want to edit files in an open PR, edit or add files to the branch and force push to the same branch!

**Comment by CodeMonk263** [comment]
Ok thanks I'll try to fix.

**Pull request closed without merging**